### ISSUE TITLE
IdsColorPicker: 3 Issues found for WCAG Guideline: 4.1.2 Name, Role, Value and 1.4.13 Content on Hover or Focus

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1037]
Describe the bug
Issue 1:  1.4.13 Content on Hover or Focus
Issue 2: 4.1.2 Name, Role, Value
Issue 3: 4.1.2 Name, Role, Value

Issue 1:
WCAG Guideline: 1.4.13 Content on Hover or Focus

Summary:  Tooltip is not provided for colors.

Step to Reproduce:
1. Launch the URL: https://main.wc.design.infor.com/ids-color-picker/example.html
2. Navigate to color picker and double tap on it
3. Verify colors for tooltip

Actual Result: Tooltip is not provided for colors.

Expect: Tooltip should be provided for colors.

Instance Observation: N/A

User Impact: If tooltip will not be provided then user will not be able to identify the colors.

Screenshot: https://www.screencast.com/t/TRlAquFnYa8

Dev Recommendations: 
1. Create a <div> element, give it a role="tooltip" and an id="IDREF". Then, give it a brief description of the action.
2. This element has to be accessible by focusing as well as hovering over. Both functionalities can be provided using CSS.
3. The tooltip has to be visually hidden using CSS.
4. Associate the tooltip with its button using aria-labelledby.

Issue 2:
WCAG Guideline: 

Summary: Expanded/ collapsed state of color picker is not defined.

Step to Reproduce:
1. Launch the URL: https://main.wc.design.infor.com/ids-color-picker/example.html
2. Navigate to color picker, hit enter on it and verify

Actual Result: Expanded/ collapsed state of color picker is not defined.

Expect Results: Screen reader should announce the expanded/ collapsed state of color picker.

Instance Observation: N/A

User Impact: If expanded/ collapsed state of hamburger menu will not be announced then screen reader user will not be able to identify the state of element.

Screenshot: https://www.screencast.com/t/dnbVdSocX2iI

Dev Recommendations: Visit this link for the code:
https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions

Issue 3:
WCAG Guideline: 4.1.2 Name, Role, Value

Summary: Name and role of colors are not defined.

Step to Reproduce:
1. Launch the URL: https://main.wc.design.infor.com/ids-color-picker/example.html
2. Navigate to color picker and hit enter
3. Navigate to colors using arrow keys and verify
    

Actual Result: Name and role of colors are not defined. Screen reader is reading it as blank.

Expect Results: Name and role of colors should be defined. Screen reader should read it as button color name for eg. button red, button black etc.

Instance Observation: N/A

User Impact: If name and role of elements will not be defined then screen reader user will not be able to understand the purpose and functionality of the element.

Screenshot: https://www.screencast.com/t/dYPAWTQ1c6n

Dev Recommendations: Provide proper aria role and aria label to colors.
Visit this link for the code:
1. https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html
2. https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html

Platform
    Windows 10
    Browser Name: Chrome

Additional context
Add any other context about the problem here. None

**Comment by tmcconechy** [comment]
https://www.nvaccess.org/download/
